pallets/parachain-staking/src/tests.rs

```diff
@@ -518,9 +518,9 @@ fn set_inflation_event_emits_correctly() {
 			annual_min: min,
 			annual_ideal: ideal,
 			annual_max: max,
-			round_min: Perbill::from_parts(57),
-			round_ideal: Perbill::from_parts(75),
-			round_max: Perbill::from_parts(93),
+			round_min: Perbill::from_parts(29),
+			round_ideal: Perbill::from_parts(38),
+			round_max: Perbill::from_parts(47),
```

Comment:
Are we okay with these new values? Just asking to ensure they are expected.

Reply:
Yes, because there are tests genesis values computed for a 12s block time, now it's 6s even in tests, so in tests rounds are 2 times faster now

Reply:
I wonder why we don't see inflation changes also in TS tests, like here https://github.com/moonbeam-foundation/moonbeam/blob/master/test/suites/dev/test-staking/test-genesis.ts#L74-L76